.travis.yml

```diff
@@ -16,13 +16,12 @@ env:
     - PYTHON=""
   matrix:
     - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
-    - CONDA_JL_VERSION="2"
 jobs:
   allow_failures:
     - julia: nightly
 
     # Missing packages on Windows
-    - env: CONDA_JL_VERSION="2"
+    - env: CONDA_JL_VERSION="3"
```

Comment:
I think this allow failure was specific to Conda 2 on Windows so this can be removed.